src/main/java/com/um/inventoryservice/BusinessLayer/StockItemServiceImpl.java

```diff
@@ -29,7 +29,7 @@ public Mono<StockItemDTO> getStockItemById(String stockItemId) {
     public Mono<StockItemDTO> insertStock(Mono<StockItemDTO> stockItemDTOMono) {
         return stockItemDTOMono
                 .map(EntityDTOUtil::toEntity)
-                .doOnNext(e -> e.setStockItemId(EntityDTOUtil.generateUUID()))
+                //.doOnNext(e -> e.setStockItemId(EntityDTOUtil.generateUUID()))
```

Comment:
maybe we don't need this generating the UUID anymore, so can take it off next time.

Reply:
thanks for the reminder, ill remove it when I get the chance